Dockerfile

```diff
@@ -5,9 +5,7 @@ COPY ./deviceauth /usr/bin/
 RUN mkdir /etc/deviceauth
 COPY ./config.yaml /etc/deviceauth/
 
-# example server private key - only for testing purpose
-# this key will be replaced with proper one later
+# mount your private key at /etc/deviceauth/rsa/private.pem
```

Comment:
The documentation is part of minio https PR https://github.com/mendersoftware/integration/pull/98 (since I already had this PR opened and had some README changes in there).